benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(STL_BENCHMARK_FLAVOR_DYNAMIC_OR_STATIC
+    static
+    CACHE STRING "Whether to use the dynamic or static standard library"
+)
+set(STL_BENCHMARK_FLAVOR_DEBUG_OR_RELEASE
+    release
+    CACHE STRING "Whether to use the debug or release library"
+)
+set(STL_BENCHMARK_FLAVOR_ITERATOR_DEBUGGING
+    default
+    CACHE STRING "What level of iterator debugging to use"
+)
+
+if(STL_BENCHMARK_FLAVOR_DYNAMIC_OR_STATIC MATCHES "^[Dd][Yy][Nn][Aa][Mm][Ii][Cc]$")

```

Comment:
Can we use @barcharcraz's casefolding technique to make this readable/searchable? Occurs below.
https://github.com/microsoft/STL/blob/17fde2cbab6e8724d81c9555237c9a623d7fb954/CMakeLists.txt#L21